src/http/http.message.ts

```diff
@@ -3,7 +3,7 @@ export class HttpMessage {
 
     constructor(
         readonly code: number,
-        readonly message: string
+        readonly message: string,
+        readonly body: string
```

Comment:
What if body not string ? Object, number, boolean and etc ?

Reply:
I guess I just miss that. I agree there is no point to force a user to convert body manually. Just unused extra action.